app/assets/javascripts/time_entries.js

```diff
@@ -3,23 +3,32 @@
 // You can use CoffeeScript in this file: http://coffeescript.org/
 
 $(function() {
-    var hideOrShowFields = function() {
-        const checked = $("#time_entry_running").is(':checked');
+  var hideOrShowFields = function() {
+    const checked = $("#time_entry_running").is(':checked');
 
-        if (checked) {
-            $('#time_entry_duration').parent().hide();
-            $('#time_entry_result').parent().hide();
-            $('#time_entry_start_time').parents(".form-group").hide();
-        } else {
-            $('#time_entry_duration').parent().show();
-            $('#time_entry_result').parent().show();
-            $('#time_entry_start_time').parents(".form-group").show();
-        }
+    if (checked) {
+      $('#time_entry_duration').parent().hide();
+      $('#time_entry_result').parent().hide();
+      $('#time_entry_start_time').parents(".form-group").hide();
+    } else {
+      $('#time_entry_duration').parent().show();
+      $('#time_entry_result').parent().show();
+      $('#time_entry_start_time').parents(".form-group").show();
     }
+  }
 
-    $(document).on("page:change", function() {
-	hideOrShowFields();
-	$('#te_datetimepicker').datetimepicker();
-	$('input[name="time_entry[running]"]').change(hideOrShowFields);
-    })
+  $(document).on("page:change", function() {
+    hideOrShowFields();
+    $('#te_datetimepicker').datetimepicker();
+    $('input[name="time_entry[running]"]').change(hideOrShowFields);
+  })
 });
+
+(function poll(){
+  setTimeout(function() {
```

Comment:
I'm using setTimeout in other projects and works fine, do you prefer setInterval instead of setTimeout?